src/jose/jwk.ts

```diff
@@ -1,47 +1,8 @@
-import * as ed25519 from './algorithms/ed25519';
-import * as secp256k1 from './algorithms/secp256k1';
-import Ajv from 'ajv';
-import type { JwkEd25519Private, JwkEd25519Public } from './algorithms/ed25519';
 import type { JwkSecp256k1Private, JwkSecp256k1Public } from './algorithms/secp256k1';
+import type { JwkEd25519Private, JwkEd25519Public } from './algorithms/ed25519';
 
-/**
- * A supported private key in JWK format.
- */
-export type JwkPrivate = JwkSecp256k1Private | JwkEd25519Private;
+export { generateKeyPair as generateSecp256k1Jwk } from './algorithms/secp256k1';
+export { generateKeyPair as generateEd25519Jwk } from './algorithms/ed25519';
 
-/**
- * A supported public key in JWK format.
- */
 export type JwkPublic = JwkSecp256k1Public | JwkEd25519Public;
-
-const jwkPublicSchema = {
-  anyOf: [
-    ed25519.jwkPublicJsonSchema,
-    secp256k1.jwkPublicJsonSchema
-  ]
-};
```

Comment:
I am not seeing code that validates `publicKeyJwk` schema in DID document. Are you saying we should kick the can and remove this for now and worry about this later? 'coz I do see a version of this need to exist somewhere when we actually have DID Document validation.

Reply:
yep! I think that we should tackle validation right after we merge the Jose PR and the blockstore PR. I didn't want to bloat this PR with validation.

Reply:
Sounds good.